shared/fs/container.tsx

```diff
@@ -89,6 +89,7 @@ const ChooseComponent = (props: ChooseComponentProps) => {
     path: props.path,
     refreshTag: Types.RefreshTag.Main,
     wantPathMetadata: true,
+    wantTlfs: true,
```

Comment:
This file is still called when we're in a browser within the Chat tab, right?

Reply:
Thanks for the early review! Nothing beats waking up in the morning with a review response :-)

This hook is called in many places, but only this one `wantsTlfs`. And this file is only used in Files tab -- it's the routable component from routes/fs.tsx.

Reply:
Oh just realized I might have mis-understood you. Yeah, if you go to a KBFS path from chat, it'd go through this.

Reply:
I mean, if I click on a KBFS link in chat, am I in the FS container? if not then we need to do something to re-fix the sync load config situation.

Reply:
oh, oops, race condition 😛 . Looks like we're on the same page.